drivers/include/adcxx1c.h

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (C) 2017 OTA keys S.A.
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup   drivers_adcxx1x ADCXX1C ADC device driver
+ * @ingroup    drivers_sensors
+ * @{
+ *
+ * @file
+ * @brief       ADCXX1C ADC device driver
+ *
+ * @author      Vincent Dupont <vincent@otakeys.com>
+ */
+
+#ifndef ADCXX1C_H
+#define ADCXX1C_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "periph/i2c.h"
+#include "periph/gpio.h"
+
+#ifndef ADCXX1C_I2C_ADDRESS
+/** ADCxx1C default address (ADCxx1C021 address) */
+#define ADCXX1C_I2C_ADDRESS  (0x54)
+#endif
+
+/**
+ * @brief   ADC resolution
+ */
+enum {
```

Comment:
@vincent-d I have a suggestion for an improved adaption to the device variants see my [PR](https://github.com/OTAkeys/RIOT/pull/7) at your repo.